src/modpacks/views.py

```diff
@@ -0,0 +1,32 @@
+from django.http import HttpResponse
+# from django.shortcuts import render
+from django.template import loader
+from modpacks.models import Modpack  # , mod
+from django.core.exceptions import ObjectDoesNotExist
+from django.http import HttpResponseNotFound
+
+
+def index(request):
+    '''
+    Lists all available modpacks
+    '''
+
+    modpacks = Modpack.objects.all()
+    template = loader.get_template('modpacks/modpacks.html')
+
+    return HttpResponse(template.render({'modpacks': modpacks}))
+
+
+def modpack(request, slug):
+    '''
+    Displays a modpack's details, including available mods and description of
+    the overall modpack
+    '''
+    try:
+        modpack = Modpack.objects.filter(slug=slug)
+    except ObjectDoesNotExist:
```

Comment:
Try to avoid using exception handling for program logic.
Is there an exists()?

Looks like it: https://docs.djangoproject.com/en/dev/ref/models/querysets/#exists